.github/workflows/staging_release.yml

```diff
@@ -0,0 +1,91 @@
+name: Deploy To Staging Server
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  build:
+    name: Webapp Prod Release
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - uses: webfactory/ssh-agent@v0.4.1
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_TV_KEY }}
+      - name: Install Dependencies
+        run: npm ci
+      - name: Check Style Lint
+        run: npm run stylelint
+      - name: Check JS Lint
+        run: npm run jslint
+      - name: Check Types
+        run: npm run tscheck
+      - name: Test
+        env:
+          GATSBY_API_TEST_USER: ${{ secrets.GATSBY_API_TEST_USER }}
+          GATSBY_API_TEST_PASS: ${{ secrets.GATSBY_API_TEST_PASS }}
+        #run: npm test -- "tradeApiClient"
+        run: echo "skipping tests"
+      - name: Install submodules
+        run: git submodule update --init
+      - name: Build
+        run: npm run build:prod
```

Comment:
I think we need to add a new command, i.e. `build:staging`, to point to the correct staging api url for example. (https://staging.zignaly.com/api ?) I can do it tomorrow if needed.

Reply:
yes we need configure the valid url for staging